### ISSUE TITLE
Possibility to define custom dartls command

### ISSUE THREAD

**Issue by andersevenrud** [akinsho/flutter-tools.nvim#34]
Since it's not recommended using lspconfig in conjunction with dartls when using this plugin, I'm wondering about situations where you have a plain Dart codebase (no Flutter) that depends on an a specific version of Dart and not the one provided in the Flutter SDK.

I'm not sure if this is a real problem (maybe more like a question of if this setup can cause any issues relating to it), but thought I'd open an issue on it anyways because this came to mind during #30 when I was messing around with settings trying to override the `dartls` cmd like this:

```
require("flutter-tools").setup {
  lsp = {
    cmd = { "some", "custom", "path", "outside", "flutter", "sdk" }
  }
}
```

**Comment by andersevenrud** [comment]
Or maybe it's a way to only use flutter-tools on flutter projects somehow via the neovim Lua APIs, and then manual lspconfig for others. I'm pretty new to all of this neovim 0.5 goodness, so there might be an obvious answer staring me right in the face.

**Comment by akinsho** [comment]
@andersevenrud it wouldn't be too hard to allow the user to pass in a custom command and use that instead of the one this plugin derives. Tbh most of the current functionality is primarily aimed at flutter usage and even then the most common usages, I work on a few dart packages and this plugin works fine with those but dart only usage isn't what I prioritise.

anyway to the original point I can make a change to allow setting a custom command for the lsp

**Comment by akinsho** [comment]
@andersevenrud this has been fixed by 791b57e.

**Comment by andersevenrud** [comment]
Sweet! Thanks :)
